apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -215,7 +240,23 @@ export default connect(
           if (typeof output === 'object') {
             output = output.toJS();
           }
-          return <Inspector key={i} data={output} />;
+          if (output.input) {
+            return <div key={i}>&gt; {output.input}</div>;
+          } else if (output.output || FALSIES.has(output.output)) {
+            return (
+              <div key={i}>
+                &lt;{' '}
+                <div style={style.inspector}>
+                  <Inspector data={output.output} />
+                </div>
+              </div>
+            );
+          } else if (
+            output.consoleLoggedOutput ||
+            FALSIES.has(output.consoleLoggedOutput)
+          ) {
+            return <Inspector key={i} data={output.consoleLoggedOutput} />;
+          }
```

Comment:
Would "output" ever have output.output and output.consoleLoggedOutput? If not, I'd suggest overloading output.output and adding a flag output.fromConsoleLogged. Then here, you can restructure the logic a bit:
```
if (output.input) {
  // keep the same
}
if (output.output || FALSIES.has(output.output)) {
  if (output.fromConsoleLog) {
    return <Inspector key={i} data={output.output} />;
  } else {
    return (
      // same <div> structure you have in 247-252
    );
  }
}
```

Reply:
Checked in with Jessie in person. Working on refactoring as suggested!

Reply:
👍